### PR TITLE
CR for Job #20394 - Build friends.js v1

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Code Review for Job #20394 - Workitem available at https://www.worklist.net/20394

### CODE REVIEW

File: interface/src/ui/overlays/TextOverlay.h [hunk 1]
```diff
@@ -55,7 +55,7 @@ class TextOverlay : public Overlay2D {
     virtual TextOverlay* createClone() const;
     virtual QScriptValue getProperty(const QString& property);
 
-    QSizeF textSize(const QString& test) const;  // Pixels
+    QSizeF textSize(const QString& text) const;  // Pixels
```
Comment (Atlante45) [inline interface/src/ui/overlays/TextOverlay.h]: Did you mean to do that?

File: examples/users.js [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        HTTP_GET_TIMEOUT = 60000,  // ms = 1 minute
+        usersRequest,
+        processUsers,
+        usersTimedOut,
+        usersTimer = null,
+        UPDATE_TIMEOUT = 5000,  // ms = 5s
+
+        MENU_NAME = "Tools",
+        MENU_ITEM = "Users Online",
+        MENI_ITEM_AFTER = "Chat...",
```
Comment (Atlante45) [inline examples/users.js]: typo

File: interface/src/ui/overlays/TextOverlay.h (hunk 1 above)
Comment (ctrlaltdavid) [inline interface/src/ui/overlays/TextOverlay.h]: Yes, it makes the parameter name match that in TextOverlay.cpp ... just a typo I noticed in passing.

**Comment by hifi-gustavo** [comment]
Build console output for commit: bbffa6474bc5a777eaf3e132c04912023a37404b available at: https://gist.github.com/e856c879eeb9ec3bfb65<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4420.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4420.dmg

File: interface/src/ui/overlays/TextOverlay.h (hunk 1 above)
Comment (Atlante45) [inline interface/src/ui/overlays/TextOverlay.h]: Nvm I'm stupid, thought you did text > test.

File: examples/users.js [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+        Overlays.editOverlay(usersHeading2D, {
+            y: viewportHeight - windowHeight + WINDOW_MARGIN_2D,
+            text: linesOfUsers.length > 0 ? "Online" : "No users online"
+        });
+    }
+
+    function requestUsers() {
+        usersRequest = new XMLHttpRequest();
+        usersRequest.open("GET", API_URL, true);
+        usersRequest.timeout = HTTP_GET_TIMEOUT * 1000;
```
Comment (Atlante45) [inline examples/users.js]: That's 16h if my math is right.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1a18decc6d003f4e224897afe17511eabf83bb6b available at: https://gist.github.com/e856c879eeb9ec3bfb65<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4420.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4420.dmg

File: examples/users.js (hunk 3 above)
Comment (ctrlaltdavid) [inline examples/users.js]: Heh, well spotted!

**Comment by Atlante45** [comment]
Just the timeout value to change, otherwise good to go.

**Comment by hifi-gustavo** [comment]
Build console output for commit: fd423e5019c72745ec756da0a0d9975875133744 available at: https://gist.github.com/e856c879eeb9ec3bfb65<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4420.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4420.dmg

**Pull request merged**